Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2577.cs

```diff
@@ -1,24 +1,24 @@
 ﻿using Xamarin.Forms.CustomAttributes;
 using Xamarin.Forms.Internals;
+using System.Linq;
 
 #if UITEST
 using Xamarin.Forms.Core.UITests;
-using Xamarin.UITest;
+using Xamarin.UITest.Queries;
 using NUnit.Framework;
 #endif
 
 namespace Xamarin.Forms.Controls.Issues
 {
-#if UITEST
-	[Category(UITestCategories.ManualReview)]
-#endif
 	[Preserve(AllMembers = true)]
 	[Issue(IssueTracker.Github, 2577, "Hamburger icon not shown when using FormsAppCompatActivity", PlatformAffected.Android)]
 	public class Issue2577 : TestMasterDetailPage
 	{
 		protected override void Init()
 		{
 			Master = new ContentPage { Title = "master page" };
+			(Master as ContentPage).Content = new ListView { AutomationId = "MasterList" };
```

Comment:
will be a little nicer
```suggestion
			Master = new ContentPage
			{
				Title = "master page",
				Content = new ListView { AutomationId = MasterList }
			};
```